wolox-training/ap-swift
# Implement Google SignIn

## Summary ##


**Trello card**: https://trello.com/c/xo0YuUjL/11-vista-de-log-in
**Trello card**: https://trello.com/c/F0x0WV8o/41-servicio-de-log-in-para-cuentas-de-google

Login view with Google-Sign-In implementation

## Test Plan ##

  * Open the app
  * Press "INGRESAR CON GOOGLE"
  * Login with a google account
  * The main tab bar view should appear after a successfull login.

## Screenshots ##

### iPhone 7

![login-view](https://cloud.githubusercontent.com/assets/25587882/22932994/d80ad820-f2a8-11e6-918e-e6be5e5889e1.png)


## Screencast ##

![google-sign-in](https://cloud.githubusercontent.com/assets/25587882/22932731/c50bc33e-f2a7-11e6-87de-ca837209f17d.gif)
